resources/js/sentry.js

```diff
@@ -32,30 +35,51 @@ let init = (app) => {
             app,
             dsn: import.meta.env.VITE_SENTRY_DSN,
             environment: import.meta.env.MODE,
-            integrations: integrations,
+            integrations: collectIntegrations(window.config.sentry.integrations),
             beforeSend: runBeforeSendMethodHandlers,
         },
         window.config.sentry.configuration
     )
 
     // Initialize Sentry
-    Sentry.init(configuration)
+    initSentry(configuration)
+}
+
+let collectIntegrations = (integrationsConfig) => {
```

Comment:
With a single integration enabled the collectIntegrations function gets compiled down to this:
```js
  collectIntegrations = (integrationsConfig) => {
    let integrations = [];
    return integrations.push(browserTracingIntegration()), integrations;
  },
```